### DISCUSSION TITLE
VMD Support?

### DISCUSSION THREAD

**Discussion by ep5000** [pixiv/three-vrm#1110] [Help]
Can VMD (MMD animations) be used on VRM models imported using this library?  If so what is the process to do this?  Thanks

**Comment by 0b5vr** [comment]
The library called "mmd-parser", created by takahirox, can be used to parse VMD files.
The data parsed by mmd-parser may be used with Three.js and three-vrm.

https://github.com/takahirox/mmd-parser

When using VMD files, please respect the MMD community and culture and check the license before using the files responsibly.
